### ISSUE TITLE
[FEAT] When setting the Workspace path instead of needing to manually hand type it, implement a FolderSuggest similar to Templater

### ISSUE THREAD

**Issue by GitMurf** [marcusolsson/obsidian-projects#30]
Right now you need to hand type the folder path for a new workspace. Would be nice to have a folder suggester helper like Templater has implemented here:

Templater Settings code: https://github.com/SilentVoid13/Templater/blob/8ceed0ad6e602089f6f0f264b889155290fd9e3e/src/settings/Settings.ts#L581-L594

Templater FolderSuggest Class: https://github.com/SilentVoid13/Templater/blob/8ceed0ad6e602089f6f0f264b889155290fd9e3e/src/settings/suggesters/FolderSuggester.ts#L6-L33

Screenshot from Templater:

![image](https://user-images.githubusercontent.com/64155612/190464436-e709c4a7-e1e6-41bb-a989-b2629f0653b5.png)

Screenshot from Projects plugin:

![image](https://user-images.githubusercontent.com/64155612/190463506-5be5dc03-05f0-4a72-858e-1ba77ea48937.png)

**Comment by marcusolsson** [comment]
Just implemented an initial version for the link field input. Will polish it and start adding places like this as well!

**Comment by GitMurf** [comment]
Awesome! Looking forward to it.

**Comment by GitMurf** [comment]
Works great @marcusolsson ! Just as I was hoping. Other than this bug that I reported here #35 which is also a bug in Templater (https://github.com/SilentVoid13/Templater/issues/830) so may just be the Suggester Class you used similar to theirs? Or maybe it is a core Obsidian problem. I just posted in Obsidian plugin-dev channel on Discord as well.

No need to re-open this issue here as can track it in #35
